.github/workflows/android.yml

```diff
@@ -57,11 +56,12 @@ jobs:
 
       - name: 💐 Install Qt
         if: ${{ steps.vars.outputs.INSTALL_QT }}
-        uses: jurplel/install-qt-action@v2
+        uses: jurplel/install-qt-action@v3
         with:
           version: 5.14.2
           modules: 'qtcharts'
-          target: ${{ steps.vars.outputs.QT_TARGET }}
+          target: 'android'
```

Comment:
```suggestion
          target: 'android'
          arch: 'android'
```

Reply:
good catch

Reply:
@m-kuhn , got the hint from this site (https://ddalcino.github.io/aqt-list-server/)